src/test/java/core/basesyntax/ReportCreatorServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportCreatorService;
+import core.basesyntax.service.impl.ReportCreatorServiceImpl;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorServiceTest {
+    private static ReportCreatorService reportCreatorService;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        reportCreatorService = new ReportCreatorServiceImpl();
+    }
+
+    @Test
+    public void createReport_initFruitsList_ok() {
+        List<Fruit> initFruitsList = initFruitList();
+        List<String> actualList = reportCreatorService.createReport(initFruitsList);
+        List<String> expectedList = List.of("fruit,quantity", "apple,20", "banana,30", "orange,55");
+        Assert.assertEquals(
+                String.format("Incorrect result of ReportList for the input:\n%s.", initFruitsList),
+                expectedList, actualList);
+        Assert.assertEquals("Incorrect size of ReportList.Size should be ",
+                expectedList.size(), actualList.size());
+    }
+
+    @Test
+    public void createReport_withoutHeadline_notOk() {
+        List<Fruit> initFruitsList = initFruitList();
+        List<String> actualList = reportCreatorService.createReport(initFruitsList);
+        List<String> expectedList = List.of("fruit,quantity", "apple,20", "banana,30", "orange,55");
+        Assert.assertEquals("Your report should start with a headline: \"fruit,quantity\" ",
+                expectedList.get(0), actualList.get(0));
+    }
```

Comment:
what difference between this and previous test cases?

Reply:
What I thought about: let's imagine somehow the createReport() method doesn't add to the final List<String> the headline. The report will be incorrect then. And here I wanted to check if the headline is added. But tests look similar, I see xD
Is it better to delete this test?